lib/docusign_dtr/query_param_helper.rb

```diff
@@ -0,0 +1,127 @@
+module DocusignDtr
+  class QueryParamHelper
+    # rubocop:disable Naming/PredicateName
+    QUERY_PARAMS = {
+      search: :search,
+      count: :count,
+      endDate: :end_date,
+      startPosition: :start_position,
+      roomStatus: :room_status,
+      ownedOnly: :owned_only,
+      transactionSide: :transaction_side,
+      isUnderContract: :is_under_contract,
+      regionId: :region_id,
+      officeId: :office_id,
+      hasSubmittedTaskList: :has_submitted_task_list,
+      hasContractAmount: :has_contract_amount,
+      sort: :sort,
+      dateRangeType: :date_range_type,
+      startDate: :start_date
+    }.freeze
+
+    class << self
+      def call(options)
+        @options = options
+        QUERY_PARAMS.each_with_object({}) do |(key, value), memo|
+          memo[key] = send(value) if @options.key? key
```

Comment:
I think this might be backwards.  

When we call this from the ruby side we should use ```{has_contract_amount: true}``` NOT ```{hasContractAmount: true}```